src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -95,8 +95,10 @@ def get(self, service_name: str) -> ServiceDescription:
     @wait_api_result
     def list_daemons(self,
                      service_name: Optional[str] = None,
+                     daemon_type: Optional[str] = None,
```

Comment:
https://github.com/ceph/ceph/blob/b745754ec29ba737f8a967c91beebb3e682c00ae/src/pybind/mgr/dashboard/controllers/host.py#L167 needs to be refactored.